pages/governance/pipeline_templates.md

```diff
@@ -0,0 +1,60 @@
+# Pipeline templates
+
+> 📘 Enterprise feature
+> Pipeline templates is only available on an [Enterprise](https://buildkite.com/pricing) plan.
+
+## Overview
+
+Pipeline templates allow you to define standard pipeline step configurations to be used by all the pipelines in your organization.
+
+When a pipeline has a pipeline template assigned to it, the step configuration for that pipeline is inherited from the template and it is no longer possible to configure a custom step configuration for that pipeline via the UI.
+
+## Creating a pipeline template
+
+Only administrators can create or update pipeline templates.
+
+1. Open your Buildkite _Organization Settings_ and choose [Templates](https://buildkite.com/organizations/-/pipeline-templates)
```

Comment:
I think we should take them right there since we have a link:

```suggestion
1. Navigate to your [organization’s pipeline template settings](https://buildkite.com/organizations/-/pipeline-templates).
```